examples/stateful-camera/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "interface"
+version = "0.1.0"
+authors = ["Michał Kawalec <michal@monad.cat>"]
+edition = "2018"
```

Comment:
This file is copied from another repo and will be cleaned up to make up for a useful example `Cargo.toml`